### PR TITLE
MWPW-124906 - Embed links inside text block

### PR DESCRIPTION
**Pull request by saugatmalla**
* Embed links inside the text block 
* Also fix loading of embed Instagram link

Resolves: [MWPW-124906](https://jira.corp.adobe.com/browse/MWPW-124906)

**Test URLs:**
- Before: https://main--blog--adobecom.hlx.page/drafts/smalla/embed-links-inside-text
- After: https://main--blog--adobecom.hlx.page/drafts/smalla/embed-links-inside-text?milolibs=embed-links

### CODE REVIEW

File: libs/blocks/instagram/instagram.js [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import { createIntersectionObserver, createTag, loadScript } from '../../utils/utils.js';
 
 export default function init(a) {
+  if (a.parentElement.firstChild.nodeType === Node.TEXT_NODE) return;
```
Comment (chrischrischris) [inline libs/blocks/instagram/instagram.js:4]: nit: move to a util in utils.js. Something like 
`const isParentText = (node) => node.parentElement.firstChild.nodeType === Node.TEXT_NODE`

**Review by chrischrischris — APPROVED** [review]
👍 One nit.  Verified in marquee text, regular text and h3 text. 

https://embed-links--milo--adobecom.hlx.page/drafts/cpeyer/embededinsta

**Review by So1ene — APPROVED** [review]

**Comment by So1ene** [comment]
One thing I noticed is the twitter link doesn't seem to work in your test URL?

**Comment by saugatmalla** [comment]
> One thing I noticed is the twitter and tiktoks don't seem to work in your test URL? image
> 
> Edit: tik tok seems to work from an incognito window

@So1ene I couldn't reproduce it. Could you provide some details?

I made some test links for each of them: 
- https://main--blog--adobecom.hlx.page/drafts/smalla/embed-tiktok
- https://main--blog--adobecom.hlx.page/drafts/smalla/embed-twitter

**Pull request merged**